src/core/service.h

```diff
@@ -104,9 +104,9 @@ typedef struct ServiceFDStore {
         Service *service;
 
         int fd;
+        bool do_poll;
         char *fdname;
         sd_event_source *event_source;
-        bool do_poll;
```

Comment:
This one however makes sense

Reply:
fix it https://github.com/systemd/systemd/pull/37424/commits/5fddc47d7e96a069c3fd0fea4452d09bd1367508